lib/Runtime/Debug/DebugContext.h

```diff
@@ -37,6 +37,9 @@ namespace Js
         // The debugger is running which means that the engine is
         // running in interpreted mode.
         Debugging,
+
+        // The debugger is running but recording is suspended.
+        RecordingSuspended,
```

Comment:
Bit mask would be good. I'm making this change on 1.6 where Kyle's change is not present. Perhaps factoring the two flags together after they are merged is a better time? @kfarnung